### DISCUSSION TITLE
Map ordinals to locations on earth

### DISCUSSION THREAD

**Discussion by casey** [ordinals/ord#797] [Ideas]
We could use S2 cells, with the cells should getting larger as the ordinals get larger. The last six ordinals, i.e., with names `a` though `f`, should map to the top level six faces of the earth.

See details of S2 here:

https://s2geometry.io/devguide/s2cell_hierarchy.html

- [ ] Are S2 cells the best choice for this? What are the alternatives?
- [ ] Propose a mapping
- [ ] Display the mapping on the ordinal page (Maybe just coordinates at first?)

**Comment by ChristopherA** [comment]
Consider looking into hilbert curves, as they have some useful privacy properties. See Twitter 🧵starting at https://twitter.com/ChristopherA/status/1243413128378892293
